packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -41,31 +48,40 @@ export class Checkbox extends BaseComponent<ICheckboxProps, ICheckboxState> impl
     };
   }
 
-  public render() {
+  /**
+   * Name of the component
+   */
+  public name(): string {
+    return 'Checkbox';
```

Comment:
this is not necessary. Can you remove this?

If you really need the name of the component, BaseComponent has a className getter that will evaluate this, for automation purposes.